components/Carousel.js

```diff
@@ -6,75 +6,92 @@ import Events from "./Events";
 
 const Carousel = () => {
   return (
-    <div className="horizontal-scroll-wrapper  rectangles">
-      <div>
-        <AboutNew />
-      </div>
-      <div>
-        <TapList />
-      </div>
-      <div>
-        <Events />
-      </div>
-      <div>
-        <Blog />
-      </div>
-      <div>
-        <Brews />
+    <div id="container">
```

Comment:
I don't think you should need these ids on anything that isn't going to be a link target.